wetterdienst/provider/dwd/radar/access.py

```diff
@@ -95,10 +97,14 @@ def collect_radar_data(
     :param start_date:      Start date
     :param end_date:        End date
     :param verify:          Whether to verify the response
+    :param read_bufr:       read bufr to pd.DataFrame
 
     :return:                ``RadarResult`` item
     """
 
+    if read_bufr:
+        import pdbufr
```

Comment:
Do you have any performance issues or annoying warnings importing this by default in the header? 
Otherwise I would recommend to import this on top.

Reply:
I think it was the warning concerning the availability of eccodes on your machine.